runtime/src/snapshot_utils.rs

```diff
@@ -2153,14 +2260,27 @@ pub fn verify_snapshot_archive<P, Q, R>(
     if let VerifyBank::NonDeterministic(slot) = verify_bank {
         // file contents may be different, but deserialized structs should be equal
         let slot = slot.to_string();
+        let snapshot_slot_dir = snapshots_to_verify.as_ref().join(&slot);
         let p1 = snapshots_to_verify.as_ref().join(&slot).join(&slot);
         let p2 = unpacked_snapshots.join(&slot).join(&slot);
-
         assert!(crate::serde_snapshot::compare_two_serialized_banks(&p1, &p2).unwrap());
         std::fs::remove_file(p1).unwrap();
         std::fs::remove_file(p2).unwrap();
+        // Bank snapshots may contain an accounts directory of hard linked append vecs,
```

Comment:
Couple of questions/comments on this block of code:
1. Seems like `accounts_hardlinks_dir` contain symlinks to the actual dirs containing hardlinks. If that's the case the naming is confusing to me.
2.  As far as I can tell, this code path (NonDeterminstic) is only called by tests. Is that the case? If not, I'm a bit concerned with all the `unwrap`ing going on here.

Reply:
I updated the PR description to explain the directories with examples.